app/sagas/init.js

```diff
@@ -26,10 +29,6 @@ const restore = function* restore() {
 		yield put(toggleMarkdown(useMarkdown));
 
 		if (!token || !server) {
-			yield all([
```

Comment:
why no equivalent for removing current server and token using realm?

Reply:
because if ```.objects('servers').filtered('currentServer = true')``` returns ```null``` haven't servers to remove.